lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -299,13 +403,13 @@ sub generate_tsvfile {
     my $r_input_file = Genome::Sys->create_temp_file_path();
 
     my @headers = (
-        'Re-sort ID',
-        'ERCC ID',
-        'subgroup',
-        'ERCC Mix',
-        'concentration (attomoles/ul)',
-        'label',
-        'count',
+        q{'Re-sort ID'},
+        q{'ERCC ID'},
+        q{'subgroup'},
+        q{'Mix 1 concentration (attomoles/ul)'},
+        q{'Mix 2 concentration (attomoles/ul)'},
+        q{'label'},
+        q{'count'},
```

Comment:
A purely cosmetic reason: I thought placing the single quotes would be easier for the end user to identify the full header titles if they ever opened the raw TSV stats file.  

When I first started this project and looked at the original TSV file I had a hard time identifying what the full column names were because of the spaces and parentheses associated with some of them.